.github/workflows/build.yml

```diff
@@ -264,6 +264,9 @@ jobs:
     - name: Read flow notebook tests with katalog
       if: ${{ ! github.event.pull_request.draft }}
       run: make run-notebook-readflow-tests-katalog
+    - name: Isolation tests with katalog
```

Comment:
if we deploy a new cluster, it wouldn't be relevant, but before calling `Isolation tests with katalog`, the `Clean cluster` should be called, otherwise installation of cert-manager fails